bindata/hypershift/ovn-kubernetes/004-config.yaml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
is it possible to make this file a symlink in the "original" ovn-k directory? I would be afraid of missing needed updates to this. Yes, the cluster name would have to be templated in. Are there any other changes?

Reply:
I created a soft link to 004-config.yaml, cno fails to open it with the following error:
```
I0315 02:19:29.300276       1 log.go:184] Failed to render: failed to render manifests: error rendering manifests: failed to read manifest bindata/hypershift/ovn-kubernetes/004-config.yaml: open bindata/hypershift/ovn-kubernetes/004-config.yaml: no such file or directory
```

Reply:
still use file, not symlink in hypershift manifest.